### DISCUSSION TITLE
error in campaign liberation with harrier

### DISCUSSION THREAD

**Discussion by jcllore** [Khopa/dcs_liberation#2709] [Q&A] [answered]
Dear colleagues, I have a problem that is happening to me since the last update, when trying to use the harrier in the liberation campaign, it gives me the following error, it did not happen before, I imagine it is due to the change in the AV weapon configuration- 8B harrier that has taken place. Can it be fixed somehow? thanks and regards
![Captura de pantalla (3)](https://user-images.githubusercontent.com/32200756/215345437-36cc346f-9aaa-4e83-94a9-22e39931d525.png)

**Comment by Starfire13** [comment, marked as answer]
Please file this as a bug report rather than Q&A, thanks!
